src/libraries/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -4,6 +4,7 @@
     <_SerializerDllIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).dll</_SerializerDllIntermediateFolder>
     <_SerializerPdbIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).pdb</_SerializerPdbIntermediateFolder>
     <_SerializerCsIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).cs</_SerializerCsIntermediateFolder>
+    <_SerializerCsAssemblyInfoIntermediateFolder>$(IntermediateOutputPath)AssemblyInfo.cs</_SerializerCsAssemblyInfoIntermediateFolder>
     <_SGenWarningText>SGEN: Failed to generate the serializer for $(AssemblyName)$(TargetExt). Please follow the instructions at https://go.microsoft.com/fwlink/?linkid=858594 and try again.</_SGenWarningText>
```

Comment:
I believe IntermediateOutputPath is a general purpose folder used by the build system in general. There is likely to be another file created at the exact same path for the main application assembly. Use a different name for this file to prevent the name clash, eg SgenAssemblyInfo.cs

Reply:
okay, fixed.